src/pocketmine/entity/Skin.php

```diff
@@ -24,6 +24,8 @@
 namespace pocketmine\entity;
```

Comment:
BC in this file is completely trashed. I don't know where to start. It doesn't look like any attempt has been made to preserve BC whatsoever.